test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Encoders/LengthPrefixedTextMessageFormatterTests.cs

```diff
@@ -42,5 +44,38 @@ public void WriteMessage(string encoded, string payload)
 
             Assert.Equal(encoded, Encoding.UTF8.GetString(output.ToArray()));
         }
+
+        private static class LengthPrefixedTextMessageWriter
```

Comment:
? Now you're testing test code

Reply:
I did not read the tests carefully. Rewriting the tests to test the encoder. Thanks for catching this.